SmallLibrary/js/main.js

```diff
@@ -0,0 +1,122 @@
+var Main = function () {
```

Comment:
The 'main' file was separated into three independent parts, they are: 
1.  TestFunctions module contains functions for testing small-lib's array functions.
2.  TestDataGenerator creates test data for test functions work.
3.  main module is responsible for data displaying source array, catching button's click events and invoking appropriate functions and displaying source and result test data.